dist/vorpal.js

```diff
@@ -990,6 +990,7 @@ vorpal._exitMode = function (options) {
   ssn._mode = false;
   this.cmdHistory.exitMode();
   ssn.modeDelimiter(false);
+  this.emit('exitmode', this.cmdHistory.peek());
```

Comment:
Can we name this `mode_exit` to fit more in line with the other event semantics?

https://github.com/dthree/vorpal/wiki/Docs-|-Events

Thanks.